src/test/OptionSettlement.t.sol

```diff
@@ -81,26 +91,112 @@ contract OptionSettlementTest is DSTest, NFTreceiver {
             expiryTimestamp: (uint64(block.timestamp) + 604800)
         });
         engine.newChain(info);
+
         // Now we have 1B DAI
         writeTokenBalance(address(this), address(dai), 1000000000 * 1e18);
         // And 10 M WETH
         writeTokenBalance(address(this), address(weth), 10000000 * 1e18);
+
         // Issue approvals
         IERC20(weth).approve(address(engine), type(uint256).max);
         IERC20(dai).approve(address(engine), type(uint256).max);
+
+        wethTotalSupply = IERC20(address(weth)).totalSupply();
+        daiTotalSupply = IERC20(address(dai)).totalSupply();
     }
 
-    function testNewChain(uint256 settlementSeed) public {
+    function testNewChain() public {
+        uint256 nextTokenId = engine._nextTokenId();
+
         Option memory info = Option({
             underlyingAsset: address(weth),
             exerciseAsset: address(dai),
-            settlementSeed: settlementSeed,
+            settlementSeed: 0,
```

Comment:
This explicitly should not be 0 for the purpose of the test. I want to make sure it gets cleared in the function, whatever the value is.

Reply:
that's why i'm writing two tests for each method. one will hard coded values and the other with fuzzing.

Reply:
Cool